### ISSUE TITLE
File changes not picked up sometimes

### ISSUE THREAD

**Issue by totakoko** [publicodes/language-server#22]
:wave: 

I regularly encounter this bug when the VS Code extensions doesn't seem to pick up changes and I have to reload the view to restart the extension

![image](https://github.com/user-attachments/assets/49f1425e-ac51-47b2-83ff-7e63645d7f56)

I can't give you the steps to reproduce the issue... :confused: 
I'm running on Fedora 40.

**Comment by EmileRolley** [comment]
The validation step is done at save to after each changes, did you try to save after you made a modification and wait for the new validation pass?

**Comment by totakoko** [comment]
Yes, I saved many times. I'll try to watch the extension output next time I get this error.
